lib/models/device.coffee

```diff
@@ -1345,6 +1345,41 @@ getDeviceModel = (deps, opts) ->
 						force: Boolean(options.force)
 		.asCallback(callback)
 
+	###*
+	# @summary Get the supervisor state on a device
+	# @name getSupervisorState
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6').then(function(state) { console.log(state); });;
+	#
+	# @example
+	# balena.models.device.getSupervisorState(123).then(function(state) { console.log(state); });;
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6', function(error) { if (error) throw error; });
```

Comment:
I'm sorry if I was confusing about the newline part. I only meant it for the `, function`.
```suggestion
	# balena.models.device.getSupervisorState('7cf02a6', function(error, state) {
		if (error) throw error;
		console.log(state);
	});
```

Reply:
Updated, thank you for clarifying!